filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -137,10 +137,10 @@ OpenGLDriver::DebugMarker::~DebugMarker() noexcept {
 
 OpenGLDriver::OpenGLDriver(OpenGLPlatform* platform) noexcept
         : DriverBase(new ConcreteDispatcher<OpenGLDriver>()),
-          mHandleArena("Handles", 2U * 1024U * 1024U), // TODO: set the amount in configuration
+          mHandleArena("Handles", FILAMENT_OPENGL_HANDLE_ARENA_SIZE_IN_MB * 1024U * 1024U), // TODO: set the amount in configuration
```

Comment:
We can probably remove that TODO comment now

Reply:
@bejado I would like to make those a runtime config option when you create the engine, someday.

Reply:
I am looking forward to having this change in the next release.  Our scenes are also potentially large and complex.
I *really* look forward to having runtime configurable options!

Reply:
Note that this is not runtime configurable with this change.